diff --git a/node_modules/react-native-flipper/android/src/release/java/com/facebook/flipper/reactnative/FlipperPackage.java b/node_modules/react-native-flipper/android/src/release/java/com/facebook/flipper/reactnative/FlipperPackage.java
new file mode 100644
index 0000000..cd94b8e
--- /dev/null
+++ b/node_modules/react-native-flipper/android/src/release/java/com/facebook/flipper/reactnative/FlipperPackage.java
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.flipper.reactnative;
+
+import com.facebook.react.ReactPackage;
+import com.facebook.react.bridge.NativeModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.uimanager.ViewManager;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Exposes the react native modules that should be created per ReactApplicationContext. Note that an
+ * application context lives shorter than the application itself, e.g. reload creates a fresh one.
+ * 
+ * This is an empty implementation to keep Flipper out of release builds while still
+ * satisfying autolink's expectation to find a FlipperPackage.
+ */
+public class FlipperPackage implements ReactPackage {
+  @Override
+  public List<NativeModule> createNativeModules(ReactApplicationContext reactContext) {
+    return Collections.emptyList();
+  }
+
+  @Override
+  public List<ViewManager> createViewManagers(ReactApplicationContext reactContext) {
+    return Collections.emptyList();
+  }
+}
\ No newline at end of file
